apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -96,88 +96,110 @@ const format = (date: Date) =>
     new Date(dayjs.utc(date).format("YYYY-MM-DDTHH:mm:ss"))
   );
 
-export const AvailabilityTab = () => {
+export const AvailabilityTab = ({ isTeamEvent }: { isTeamEvent: boolean }) => {
   const { t, i18n } = useLocale();
   const { watch } = useFormContext<FormValues>();
 
+  const EventTypeSchedule = () => {
```

Comment:
Move it to a separate component, so that it can be conditionally put inside a toggle or outside it 

There aren't any changes inside this component. Use 'Hide whitespace' option to see lesser changes.